examples/rpc-transport-throttled/src/example.ts

```diff
@@ -0,0 +1,137 @@
+/**
+ * EXAMPLE
+ * Create a custom RPC transport that will only send a certain number of requests per second.
+ *
+ * Before running any of the examples in this monorepo, make sure to set up a test validator by
+ * running `pnpm test:live-with-test-validator:setup` in the root directory.
+ *
+ * To run this example, execute `pnpm start` in this directory.
+ */
+import { createLoggerWithTimestamp } from '@solana/example-utils/createLogger.js';
+import {
+    ClusterUrl,
+    createDefaultRpcTransport,
+    createSolanaRpcFromTransport,
+    mainnet,
+    RpcTransportFromClusterUrl,
+} from '@solana/web3.js';
+
+const log = createLoggerWithTimestamp('Throttling transport');
+
+/**
+ * SETUP: Define the maximum requests-per-second this RPC should make.
+ */
+const MAX_RPS = 2;
+
+/**
+ * SETUP: CUSTOM THROTTLING TRANSPORT
```

Comment:
nit: Is this a "SETUP" phase if this is what the example is supposed to highlight? IMO "SETUP" sections should be reserved for stuff we need to prepare for the main star of the show to work. In other words, they could be hidden and the examples would still make senses.